src/main/flight/mixer.c

```diff
@@ -572,12 +574,26 @@ static void calculateThrottleAndCurrentMotorEndpoints(timeUs_t currentTimeUs)
             pidResetIterm();
         }
     } else {
+        static float motorRangeMinIncrease = 0;
```

Comment:
Can you please put the static local variable definition at the beginning of the function definition?

Reply:
done